libcodechecker/orm_model.py

```diff
@@ -116,63 +116,59 @@ def __init__(self, filepath, content_hash):
 class BugPathEvent(Base):
     __tablename__ = 'bug_path_events'
 
-    id = Column(Integer, autoincrement=True, primary_key=True)
     line_begin = Column(Integer)
     col_begin = Column(Integer)
     line_end = Column(Integer)
     col_end = Column(Integer)
+
+    order = Column(Integer, primary_key=True)
+
     msg = Column(String)
     file_id = Column(Integer, ForeignKey('files.id', deferrable=True,
```

Comment:
We should decide when there events will be automatically deleted. Right now cascade delete tries to remove this if the corresponding file is removed or the corresponding report is removed. Is it possible that there is a conflict at deletion because of the foreign keys?